docs/data/base/components/modal/modal.md

```diff
@@ -126,6 +126,14 @@ In order to display an Unstyled Modal rendered on the server, you need to disabl
 
 {{"demo": "ServerModal.js", "defaultCodeOpen": false}}
 
+## Position fixed elements
+
+The modal disables the page scrolling while open by setting the `overflow: hidden` CSS property on the relevant scroll container.
```

Comment:
👍 nice I remember this mentioning on some pull request. It's nice to have now link to share.